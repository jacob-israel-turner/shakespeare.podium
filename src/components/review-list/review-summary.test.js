import React from 'react'
import renderer from 'react-test-renderer'
import ReviewSummary from './review-summary'
import {StyleSheetTestUtils} from 'aphrodite'

beforeAll(() => {
  StyleSheetTestUtils.suppressStyleInjection()
});

afterAll(() => {
  StyleSheetTestUtils.clearBufferAndResumeStyleInjection()
});

describe('<ReviewSummary />', () => {
  const fakeId = 123

  it('renders without crashing', () => {
    const component = renderer.create(<ReviewSummary id={fakeId} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
    expect(tree.children[0].children[0]).toBe('LOADING') // TODO: This is fragile.  Re-write with Enzyme.
  });
})
