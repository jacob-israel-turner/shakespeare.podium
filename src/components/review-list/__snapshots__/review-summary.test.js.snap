exports[`<ReviewSummary /> renders without crashing 1`] = `
<tr
  className="container_120drhm">
  <td>
    LOADING
  </td>
</tr>
`;
